sdk/inc/azure/iot/az_iot_hub_client.h

```diff
@@ -46,6 +46,8 @@ typedef struct az_iot_hub_client_options
   az_span module_id; /**< The module name (if a module identity is used). */
   az_span user_agent; /**< The user-agent is a formatted string that will be used for Azure IoT
                          usage statistics. */
+  az_span model_id; /**< The model id used to identify the capabilities of a device based on the

```

Comment:
`model id` is usually referred to as `ModelId` in English at least from what I've tracked.

Reply:
unless reallllly necessary I'll probably keep as is to match the above module name